### PR TITLE
Update our-team.tsx

### PR DESCRIPTION
**Pull request by RymMichaut**
Update Rym's job title

### CODE REVIEW

File: src/components/pages/about-us/our-team/our-team.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import Image from 'next/image';
+of Digital Communication & Marketingimport Image from 'next/image';
```
Comment (FabienLelaquais) [inline src/components/pages/about-us/our-team/our-team.tsx:1]: Missing carriage return, at least,  don't you think?

**Review by FabienLelaquais — CHANGES_REQUESTED** [review]
Small hickup there...

File: src/components/pages/about-us/our-team/our-team.tsx (hunk 1 above)
Comment (RymMichaut) [reply src/components/pages/about-us/our-team/our-team.tsx:1]: indeed. The correct one would start with "Head"

**Review by FabienLelaquais — APPROVED** [review]
👌

**Pull request merged**